__tests__/app.test.js

```diff
@@ -340,4 +340,31 @@ describe('/api/articles/:article_id', () => {
             expect(response.body.msg).toBe('Bad request')
         })
     })
+})
+
+describe('DELETE /api/comments/:comment_id', () => {
+    test('GET:204 Should delete given comment by commentID', () => {
```

Comment:
None of these are GET requests they are DELETE request so make sure descriptions match the endpoints